### PR TITLE
go solution with tests

### PR DESCRIPTION
**Pull request by drkennetz**
```
C:\Users\dkennetz\fun\CodingDojo\challenges\2021-06-01-find-three-largest-numbers\solutions\go\drkennetz>go test -bench=.
SUCCESS!
SUCCESS!
SUCCESS!
goos: windows
goarch: amd64
pkg: solution
BenchmarkFindThreeLargest-6     23541135                44.8 ns/op
PASS
ok      solution        1.192s
```

### CODE REVIEW

**Review by birenderjit — APPROVED** [review]
Nice use of the math.MaxInt32 in the result arrays.

**Comment by drkennetz** [comment]
@birenderjit I appreciate that bro, I realized they actually have a MinInt32 as well (-maximum), but I had already submitted the PR 🗡️.

File: challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+findThreeLargestNumbers([11, -7, 5]) -> [-7, 5, 11]
+findThreeLargestNumbers([1]) -> Null
+```
+ */
+
+func FindThreeLargest(i []int) []int {
+	if len(i) < 3 {
+		return nil
+	}
+	result := []int{-math.MaxInt32, -math.MaxInt32, -math.MaxInt32}
```
Comment (ericwburden) [inline challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz.go:33]: Neat!

File: challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz_test.go [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+	tables := []struct {
+		input []int
+		result []int
+	}{
+		{[]int{141, 2, 17, -7, -17, -27, 18, 541, 8, 7, 7}, []int{18, 141, 541}},
+		{[]int{11, -7, 5, -7}, []int{-7, 5, 11}},
+		{[]int{1}, nil},
+	}
+
+	for _, table := range(tables) {
```
Comment (ericwburden) [inline challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz_test.go:19]: So, `Go::range()` is basically `Python::enumerate()`?

**Review by ericwburden — COMMENTED** [review]
Love seeing these testing setups!

**Review by ericwburden — APPROVED** [review]
Gah, forgot to select `Approve`.

File: challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz_test.go (hunk 2 above)
Comment (drkennetz) [reply challenges/2021-06-01-find-three-largest-numbers/solutions/go/drkennetz/drkennetz_test.go:19]: yeah exactly like enumerate!

**Pull request merged**